akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingSpec.scala

```diff
@@ -130,6 +130,11 @@ abstract class ClusterShardingSpecConfig(
    * mode, then leverage the common config and fallbacks after these specific test configs:
    */
   commonConfig(ConfigFactory.parseString(s"""
+    akka.loglevel = "DEBUG"
+    
+    akka.cluster.sharding.verbose-debug-logging = on
+    akka.loggers = ["akka.testkit.SilenceAllTestEventListener"]
```

Comment:
I think this will be pretty confusing when debugging test failures. Will logs from all nodes be flushed (and included in output) before it's too late? The order of the log entries will probably be chunked by node but can still be mixed together? We would then need some tool to reassemble in timestamp order or strictly separate them by node?

Related https://github.com/akka/akka/issues/27634

Reply:
You get the JVM name in the logs so it is straightforward to take failure logs and get a separate log file per node.

Reply:
ok, and what about the flushing from all nodes?

Reply:
I think that's worth adding but for most tests they'll fail anyway due to a failed barrier